src/App.js

```diff
@@ -17,9 +17,42 @@ import AddTodoForm from "./AddTodoForm";
 function App() {
   const [todoList, setTodoList] = React.useState([]);
   const [isLoading, setIsLoading] = React.useState(true);
+  const url = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${process.env.REACT_APP_TABLE_NAME}`;
+
```

Comment:
Nitpick:
YES! I love that you've defined this URL as a reusable constant! However, one recommendation would be to define it above the `App` function with UPPERCASE letters. You'll often see this pattern to denote that this variable is statically defined and should never be changed:
```diff
+ const AIRTABLE_API_URL = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${process.env.REACT_APP_TABLE_NAME}`;

function App() {
  ...etc
- const url = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${process.env.REACT_APP_TABLE_NAME}`;
  ...etc
}
```